IceCastBar.lua

```diff
@@ -23,6 +23,19 @@ if IceHUD.WowClassic then
 	UnitChannelInfo = ChannelInfo
 end
 
+-- Fulzamoth 2019-09-27 : Use LibClassicCasterino if it's there so we can use TargetCast 
+--                        module in Classic WoW
+if IceHUD.WowClassic then
+	LibClassicCasterino = LibStub("LibClassicCasterino", true)
+	UnitCastingInfo = function(unit)

```

Comment:
Add validation of `LibClassicClasterino` so things don't fail if the lib isn't present.

Reply:
Sure. It's checked everywhere it's referenced, but I can add here too.

Reply:
You're right, this entire module won't be loaded if the lib isn't present, so it shouldn't be an issue. My bad.